### ISSUE TITLE
Super + D

### ISSUE THREAD

**Issue by spheras** [spheras/desktopfolder#48]
Super + D ('D' from Desktop) is a shortcut installed by desktopfolder the first time you launch the app.. if it has not conflict in your system with other shortcut.  I'm afraid Super + D will be a system Shortcut for Elementary Juno. 

https://github.com/donadigo/gala-pip-plugin

Pip plugin initially used Super+D shortcut, so it will be probably used in gala as well. Currently if the Shortcut have conflicts, simply the app doesn't install the shortcut. We could select a different strategy, like ask to the user the shortcut to be installed, or have a set of keys (and let it know to the user someway the shortcut)... The user can go always to the keyboard configuration and change the shortcut.

On the other hand, currently the "Show Desktop" function is done by Wnck, which will not work in Wayland afaik... I mean, maybe we will not need the shorcut for Juno :rofl:

**Comment by aljelly** [comment]
How about Super + Alt + D?

**Comment by spheras** [comment]
Yes, we have other alternatives, but the question still remains.. It is basically.. what to do if the user is using the shortcut we decide (i.e. Super + Alt + D)? Alternatives:

1. Don't install nothing (is what desktopfolder do now when a conflict is found)
2. Try others shortcuts (from a list we decide) (the question is, how we can let it know to the user)
3. Ask to the user the shortcut

**Comment by aljelly** [comment]
I just reinstalled elementary and I'm trying the elementary daily PPA. It looks like Super + F is actually the PIP shortcut. If this stays as the PIP shortcut it may not be necessary to change the Show Desktop shortcut.

**Comment by spheras** [comment]
Ok. In my opinion we sould wait and see what happens with this. At least for the next release 1.0.4

**Comment by eljefuri** [comment]
@spheras What is the argument against allowing the user to define this shortcut, but also including some sensible default?

**Comment by spheras** [comment]
well, the argument is just the complexity to do that. In my opinion, the ideal situation should be:
- if the default shortcut is taken already, ask to the user for any other
- In any case add at the configuration window a way to change that key.

As I say, the point is just the complexity.. you know, trying always to focus on the most important and requested features due to the short free time we have to accomplish this.

**Comment by spheras** [comment]
@fossfreedom, could you check how this shortcut is being installed in budgie?  You can find the way this shortcut is installed in Application.vala, createshortcut method.  I'm not sure what is happening there, if the shortcut is being installed or not  (due to you have already a shortcut like that by default).
